examples/user_guide/Plotting_with_Bokeh.ipynb

```diff
@@ -695,7 +714,7 @@
     "\n",
     "points = hv.Points(\n",
     "    elements, ['electronegativity', 'density'],\n",
-    "    ['name', 'symbol', 'metal', 'CPK', 'atomic radius']\n",
+    "    ['name', 'symbol', 'metal', 'CPK', 'atomic radius'],\n",
```

Comment:
I think we should move the section from Bokeh.models import HoverTool` to the end of this section. It is the last resort when things do not work.

However, we should at least mention early that it uses the model behind the scenes and links to Bokeh's documentation.